package.json

```diff
@@ -69,5 +72,12 @@
     "plugins": [
       "transform-class-properties"
     ]
+  },
+  "lint-staged": {
+    "src/**/*.js": [
+      "prettier --print-width=120 --single-quote --trailing-comma=all --write",
```

Comment:
I think we can only specify the actual width here, cannot opt-out of this - except ofc by passing rly big int or maybe even Infinity? But i aint sure if that works.

Point of prettier is to bring consistency and not every1 is using vim/emacs or have their other editor configured like this